lib/internal/errors.js

```diff
@@ -129,6 +129,8 @@ E('ERR_INVALID_OPT_VALUE',
   (name, value) => {
     return `The value "${String(value)}" is invalid for option "${name}"`;
   });
+E('ERR_INVALID_REPL_EVAL_CONFIG',
+  'Cannot specify both breakEvalOnSigint and eval for REPL');
```

Comment:
Thank you for fixing it quickly! As I said before, this will (sadly) make this PR semver-major, meaning that we probably won't be able to release it before node 9.